### PR TITLE
Updating Pulsar component to new ring style

### PR DESCRIPTION
**Pull request by dalvarez**
This will update the Gestalt Pulsar component to the new blue ring version. 
Activation tested. PDS approved.

### CODE REVIEW

**Comment by christianvuerings** [comment]
Deploy preview for *gestalt* ready!

Built with commit 26e4aaa1f9390a219327845aaa5c6a045ddd23a6

https://deploy-preview-115--gestalt.netlify.com

File: packages/gestalt/src/Pulsar/Pulsar.css [hunk 1]
```diff
@@ -1,86 +1,49 @@
...
-.outerCircle {
-  composes: absolute from "../Layout.css";
-  composes: circle from "../Borders.css";
-  composes: darkGrayBg from "../Colors.css";
-  animation-duration: 3s;
-  animation-iteration-count: inherit;
-  animation-name: OuterAnimation;
-  height: 100%;
-  width: 100%;
+.innerCircle:hover {
```
Comment (christianvuerings) [inline packages/gestalt/src/Pulsar/Pulsar.css]: @dalvarez should we also add a `:focus` here with the same styles? How does this behave when you're navigating with the keyboard only

**Comment by christianvuerings** [comment]
@dalvarez Could we also update `CHANGELOG.md`?

**Comment by dalvarez** [comment]
@chrislloyd good callout! i removed the animation start delay
@christianvuerings going to add the focus and keyboard support, as well as another example in the Pulsar doc

**Pull request merged**